PR for SRVLOGIC-300, SRVLOGIC-190, SRVLOGIC-188, SRVLOGIC-192: Document "Serverless Logic" parent directory for Openshift Serverless Logic docs. (#76490)
openshift/openshift-docs: modules/serverless-logic-workflow-base-builder-image.adoc

// Module included in the following assemblies:
//
// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc

:_content-type: PROCEDURE
[id="serverless-logic-workflow-base-builder-image_{context}"]
= Configuring the workflow base builder image

> If your scenario requires strict policies for image usage, such as security or hardening constraints, you can replace the default image used by the {ServerlessLogicOperatorName} to build the final workflow container image.

Author: I have added a note about this in the description. 
{ServerlessLogicOperatorName} will not reflect right now because I have added this attribute in the (https://github.com/openshift/openshift-docs/pull/76488). Once we merge that PR this attribute will reflect correctly.

> If your scenario requires strict policies for image usage, such as security or hardening constraints, you can replace the default image used by the {ServerlessLogicOperatorName} to build the final workflow container image.

Author: {ServerlessLogicOperatorName} contains the word Operator so I will not mention it here.

> By default, the {ServerlessLogicOperatorName} uses the image distributed upstream to build workflows. To change this image, edit the SonataFlowPlatform custom resource in the namespace where you deployed your workflows:

Reviewer: ```suggestion
By default, the {ServerlessLogicOperatorName} uses the image distributed in the official Red Hat Registry to build workflows. To change this image, edit the SonataFlowPlatform custom resource in the namespace where you deployed your workflows:
```

> By default, the {ServerlessLogicOperatorName} uses the image distributed in the official Red Hat Registry to build workflows. To change this image, edit the SonataFlowPlatform custom resource in the namespace where you deployed your workflows:

Author: {ServerlessLogicOperatorName} contains the word Operator so I will not mention it here.

.Prerequisites

* {ServerlessLogicOperatorName} is installed on your cluster.
* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
* You have created your {ServerlessLogicProductName} project
* Install the OpenShift CLI `(oc)`.

.Procedure

. Get the `SonataFlowPlatform` Name.
+

> Use the following command to list all SonataFlowPlatform resources in your namespace:

Reviewer: ```suggestion
Use the following command to list all `SonataFlowPlatform` resources in your namespace:
```

+
[source,terminal]
----
$ oc get sonataflowplatform -n <your_namespace>
----
+ Replace `<your_namespace>` with the name of your namespace.

> . Patch the SonataFlowPlatform with the new builder image.

Reviewer: ```suggestion
. Patch the `SonataFlowPlatform` with the new builder image.
```

// Module included in the following assemblies:
//
// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc

:_content-type: PROCEDURE
[id="serverless-logic-workflow-base-builder-image_{context}"]
= Configuring the workflow base builder image

If your scenario requires strict policies for image usage, such as security or hardening constraints, replace the default image used by the {ServerlessLogicOperatorName} to build the final workflow container image.

By default, the {ServerlessLogicOperatorName} uses the image distributed in the official Red{nbsp}Hat Registry to build workflows. To change this image, edit the `SonataFlowPlatform` custom resource (CR) in the namespace where you deployed your workflows:

.Prerequisites

* You have the {ServerlessLogicOperatorName} installed on your cluster.
* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.

> * You have created your {ServerlessLogicProductName} project

Author: Done!
